core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -63,7 +63,7 @@ class DAGScheduler(
     mapOutputTracker: MapOutputTrackerMaster,
     blockManagerMaster: BlockManagerMaster,
     env: SparkEnv,
-    clock: Clock = SystemClock)
+    clock: org.apache.spark.util.Clock = SystemClock)
```

Comment:
Spark has a few different `Clock` classes, so maybe this was to disambiguate: https://issues.apache.org/jira/browse/SPARK-4682